Gemfile

```diff
@@ -199,6 +199,8 @@ group :development, :test do
   gem "chromedriver-helper"
   gem 'simplecov', require: false, group: :test
   gem 'selenium-webdriver', "~> 3.142.0"
+
+  gem 'dotenv-rails'
```

Comment:
Would you add a version to the Gemfile? If i recall correctly, the security team wants hardcoded versions to prevent an accidental `bundle update` from introducing a security flaw.